src/css/controls/imports/settings-menu.less

```diff
@@ -0,0 +1,61 @@
+@import "../../shared-imports/vars";
+
+.jw-settings-menu {
+    display: none;
+    position: absolute;
+    z-index: -1;
+    flex-flow: column wrap;
+    align-items: flex-start;
+    bottom: @controlbar-height;
+    right: 0;
+    height: ~"calc(100% - @{controlbar-height})";
```

Comment:
I did this because it makes more sense from an organizational standpoint; and because we'd be applying less styles to mobile devices, which have less processing power.